thorlcr/activities/wuidwrite/thwuidwrite.cpp

```diff
@@ -280,7 +280,7 @@ class CWorkUnitWriteLocalActivityMaster : public CWorkUnitWriteMasterBase
 
 CActivityBase *createWorkUnitWriteActivityMaster(CMasterGraphElement *container)
 {
-    if (container->queryLocalOrGrouped())
+    if (container->queryOwner().isLocalChild())
```

Comment:
I think this change is correct, CWorkUnitWriteLocalActivityMaster and it's counterpart in the slave (CWorkUnitWriteLocalSlaveActivity), is used in the context of a CQ, where the partial results are streamed differently.

But prior to this change, it was also being used if the activity was marked 'grouped' when it should not have been.